File: C_Sharp_code/Example/WindowsFormsApp/CS_demo21/Television.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace CS_demo21
{
    public abstract class Television
    {
        public abstract void increaseVolume();

        public abstract void decreaseVolume();

        public void turnOff()
        {
            Console.WriteLine("television off");
        }

        public void turnOn()
        {
            Console.WriteLine("television on");
        }
    }
}

File: C_Sharp_code/Example/WindowsFormsApp/VB_demo22/Television.vb
Public MustInherit Class Television
    Public MustOverride Sub IncreaseVolume()

    Public MustOverride Sub DecreaseVolume()

    Public Sub TurnOff()
        Console.WriteLine("television off")
    End Sub

    Public Sub TurnOn()
        Console.WriteLine("television on")
    End Sub

End Class
